File: Modul 7/Soal 5/PRAK605-2210817120012-Noorlaini Marratain.c
#include <stdio.h>

int main()
{
        int matriksA[10][10], matriksB[10][10], matriksAXB[10][10];
        int x, y, a, b, jumlah=0;
        scanf("%d", &b);
        printf("Matriks A\n");
        for(x=0; x<b; x++){
            for(y=0; y<b; y++){
                scanf("%d", &matriksA[x][y]);
            }
        }
        printf("Matriks B\n");
        for(x=0; x<b; x++){
            for(y=0; y<b; y++){
                scanf("%d", &matriksB[x][y]);
            }
        }
        for(x=0; x<b; x++){
            for(y=0; y<b; y++){
                for(a=0; a<b; a++){
                jumlah = jumlah + matriksA[x][a]*matriksB[a][y];
            }
            matriksAXB[x][y] = jumlah;
            jumlah=0;
        }
    }
    printf("Matriks AXB\n");
    for(x=0; x<b; x++){
        for(y=0; y<b; y++){
            printf("%d ", matriksAXB[x][y]);
        }
        printf("\n");
    }
    return 0;
}

File: Modul 7/Soal 5/PRAK605-2210817120012-Noorlaini Marratain.py
n = int(input())
matriksA = []
print('Matriks A')
for i in range (n):
    k = list(map(int, input().split()))
    matriksA.append(k)
matriksB = []
print('Matriks B')
for i in range (n):
    k = list(map(int, input().split()))
    matriksB.append(k)
matriksAXB = []
print('Matriks AXB')
for i in range (n):
    matriksAXB.append([])
    for j in range (n):
        x = 0
        for k in range (n):
            x = x + matriksA[i][k] * matriksB[k][j]
        matriksAXB[i].append(x)
for baris in matriksAXB:
    for k in baris:
        print(k, end=' ')
    print()
        
